apps/web/src/services/asset-inventory/cloud-service/modules/cloud-service-list/CloudServiceListCard.vue

```diff
@@ -190,15 +190,20 @@ export default {
 
                 .service-type-name {
                     @apply text-sm text-gray-600;
-                    max-width: 80%;
+                    max-width: 90%;
                     font-weight: 400;
                     line-height: 1.25;
-                    display: inline-block;
+                    display: -webkit-box;
+                    -webkit-line-clamp: 2;
+                    -webkit-box-orient: vertical;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    white-space: normal;
```

Comment:
nitpick:You can use Tailwind's truncate!

Reply:
```
                    @apply text-sm text-gray-600 truncate;
                    max-width: 90%;
                    font-weight: 400;
                    line-height: 1.25;
                    display: -webkit-box;
                    -webkit-line-clamp: 2;
                    -webkit-box-orient: vertical;
                    //overflow: hidden;
                    //text-overflow: ellipsis;
                    white-space: normal;
                    height: calc(100% + 0.5rem);
```

This works! Thanks for tip